chore: move server scripts to dashboard/

parthsidpara opened the pull request:
Move `index.ts` and `deno.json` into `dashboard/` from root.

parthsidpara committed:
move server scripts to dashboard/

debloper commented:
Haven't taken a deep look at it yet, but can you validate if the binary build CI is fine with this change?

parthsidpara committed:
upgrade index file path in build workflow

parthsidpara committed:
place binary into dashboard/ folder

update setup.sh and piosk-dashboard.template to place piosk binary into dashboard/

parthsidpara commented:
I've committed two changes:
1. Updated the path to the `index.ts` file in the build workflow
2. Updated `setup.sh` to place the downloaded binary into `/dashboard` folder and adjust its path in `piosk-dashboard.template`

lmk if this works, or if you’d prefer to keep the binary in the root directory, or if any other changes are needed.

parthsidpara committed:
retract output path in deno.json

debloper reviewed (approved):
Ideally it's preferred to keep the trailing newline, and also to keep the file hash intact; but that ship has sailed, and it's functionally okay... so let's go with it.

<img width="711" height="561" alt="Image" src="https://github.com/user-attachments/assets/624c67c6-0b2c-487f-a00b-daeb558dfc7e" />

Also:
- https://stackoverflow.com/questions/729692/
- https://stackoverflow.com/questions/5813311/
